console-plugins/dense-mapping-constraints-plugin/include/dense-mapping/dense-mapping-parallel-process.h

```diff
@@ -23,19 +23,24 @@ size_t parallelProcess(
     // Nothing todo.
     return 0u;
   }
+
+  const size_t num_processing_threads = std::min(num_elements, num_threads);
+  VLOG(1) << "num_processing_threads vs num_threads: " << num_processing_threads
```

Comment:
Can we increase verbosity here?

Reply:
that was just debug actually -> I removed it